log_outgoing_requests/formatters.py

```diff
@@ -1,30 +1,39 @@
 import logging
 import textwrap
 
+from django.conf import settings
+
 
 class HttpFormatter(logging.Formatter):
     def _formatHeaders(self, d):
         return "\n".join(f"{k}: {v}" for k, v in d.items())
 
+    def _formatBody(self, content: dict, request_or_response: str) -> str:
+        if settings.LOG_OUTGOING_REQUESTS_LOG_BODY_TO_STDOUT:
```

Comment:
this setting name is confusing, it should be something like `LOG_OUTGOING_REQUESTS_LOGGING_MODULE_EMIT_BODIES`

* it focuses on the stdblib `logging` context
* it captures that both request/response bodies are logged

While logging in containers is configured to stdout, you can also configure it to log to file or even a networked service, so the setting name is just wrong if you write to file for example.